docs/getting-started/integrations/angular.md

```diff
@@ -227,13 +235,22 @@ export class AppComponent {
 			Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo
 		],
 		licenseKey: '<YOUR_LICENSE_KEY>',
-		mention: {
-			// Mention configuration
-		}
+		// mention: {
+		//     // Mention configuration
```

Comment:
This looks somewhat weir, the double comment. Wouldn't one be enough?

Reply:
Right, it is excessive - a single comment is enough.